src/PatientChecking/PatientChecking/Controllers/PatientController.cs

```diff
@@ -1,21 +1,64 @@
 ﻿using Microsoft.AspNetCore.Mvc;
+using PatientChecking.Services.Repository;
+using PatientChecking.Views.ViewModels;
 using System;
 using System.Collections.Generic;
+using System.Dynamic;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace PatientChecking.Controllers
 {
     public class PatientController : BaseController
     {
-        public IActionResult Index()
+
+        private readonly IPatientService _patientService;
+
+        public PatientController(IPatientService patientService)
         {
-            return View();
+            _patientService = patientService;
+        }
+
+        public async Task<IActionResult> Index()
+        {
+            PagingRequest request = new PagingRequest()
+            {
+                PageIndex = 1,
+                PageSize = 10,
+                SortSelection = 0
+            };
+            PagedResult<PatientListViewModel> pagedResult = await _patientService.GetListPatientPaging(request);
+            dynamic mymodel = new ExpandoObject();
```

Comment:
Should create a view model instead of using `ExpandoObject()`. Please check for other places.

Reply:
This has been applied in code.